Traducir.Web.Net/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -0,0 +1,18 @@
+<environment include="Development">
+    <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
+    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+</environment>
+<environment exclude="Development">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.17.0/jquery.validate.min.js"
```

Comment:
I guess this came with the bootstrap code from .net, but I don't want to use client side validation anywhere

Reply:
This too was generated automatically by Visual Studio and most likely can just go away as part of the cleanup process.